trin-core/src/portalnet/protocol.rs

```diff
@@ -178,6 +179,16 @@ impl PortalnetEvents {
                 }
                 Err(e) => panic!("Unable to respond to FindContent: {}", e),
             },
+            // ACCEPT - in response to an offer msg, requests the data for given content-key
+            // this demonstrates the recipients willingness to store the data
+            Request::Offer(Offer { content_keys }) => {
+                // by default, we accept to store data
+                let connection_id = vec![9,9];
```

Comment:
Probably worth commenting that this `vec![9,9]` is just a placeholder until uTP is implemented and used to generate and send a connection ID.